README.md

```diff
@@ -246,6 +248,8 @@ Translations of the guide are available in the following languages:
   As far as embedded expressions go, there are also two acceptable
   options:
 
+  **Skookum uses the first option, no spaces in string embedded expressions.**
```

Comment:
I feel that this should only apply to spaces before and after the brackets (contradictory to the rule above, but following the general spacing rule) -- for example:

```ruby
puts "Tomorrow is #{Date.today + 1}"
#vs
puts "Tomorrow is #{Date.today+1}"
```

I guess it could be argued that `tomorrow` deserves to be computed outside of the string interpolation, but this is just a contrived example.